modules/sqs/main.tf

```diff
@@ -25,6 +34,21 @@ data "aws_iam_policy_document" "status_update_queue_policy" {
 }
 
 
+## metadata_queue - copy_to_glacier writes to this database update queue.
+## ==============================================================================
+resource "aws_sqs_queue" "metadata_queue" {
+  ## OPTIONAL
+  name                        = "${var.prefix}-metadata-queue.fifo"
```

Comment:
wondering if we should include orca in the name to better identify that this is an orca queue. Thoughts?

Reply:
I think we're fine. When we pass it out of our ORCA space we call it the `orca_metadata_queue` and we have plenty of resources that don't have an `orca` prefix.